sonar-plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -264,8 +264,16 @@ public NewAction setDeprecatedKey(@Nullable String s) {
       return this;
     }
 
-    public NewAction setDescription(@Nullable String s) {
-      this.description = s;
+    /**
+     * Used in Orchestrator
+     */
+    public NewAction setDescription(@Nullable String description) {
+      this.description = description;
+      return this;
+    }
+
+    public NewAction setDescription(@Nullable String description, Object... descriptionArgument) {
```

Comment:
Interesting ! You've just forgot to add the @since annotation on this method and on the last one, as this file is in the API.